core/app/models/spree/payment.rb

```diff
@@ -144,6 +144,20 @@ def credit_allowed
       amount - (offsets_total.abs + refunds.sum(:amount))
     end
 
+    # The total amount this payment can be credited in cents.
+    #
+    # For currencies which do not support decimal places, it will return the
+    # full value.
+    #
+    # @return [Fixnum] The amount of this payment minus offets and refunds
+    #   in cents.
+    def credit_allowed_in_cents
```

Comment:
Whats the rational here for returning the in_cents rather than the money object and letting the caller do with it as it wishes?